docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -105,12 +105,13 @@ browsers(browser) → this
-String                                    | The path to the browser executable.                                                                                                                                   | Local browser
-`{path: String, cmd: String}`             | The path to the browser executable (`path`) with command line parameters (`cmd`).                                                                                     | Local browser
-[BrowserConnection](browserconnection.md) | The remote browser connection.                                                                                                                                        | Remote browser
+Type                                                                                                 | Description                                                                                                                                                           | Browser Type
+---------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------
+String                                                                                               | The browser name. For the full list of names, see [Supported Browsers](../common-concepts/using-browsers.md#supported-browsers) topic.                                            | Local browser
+`{path: String}`                                                                                     | The path to the browser executable.                                                                                                                                   | Local browser
+`{path: String, cmd: String}`                                                                        | The path to the browser executable (`path`) with command line parameters (`cmd`).                                                                                     | Local browser
+[BrowserConnection](browserconnection.md)                                                            | The remote browser connection.                                                                                                                                        | Remote browser
+`{$browser-provider-name: String}`                                                                   | The browser alias that consists of the browser provider name and the name of a browser itself.                                                                        | A browser introduced by a [browser provider plugin](../common-concepts/using-browsers.md#browser-provider-plugins).
```

Comment:
M-m, this looks a bit weird. From my POV it looks like I must use  `runner.browsers({ "$browser-provider-name": "saucelabs:chrome" })` to start Chrome on SauceLabs. Actually, you just need to pass the string to `browsers()` method: `runner.browsers("saucelabs:chrome")`